pkg/indexer/storer/groupMessage.go

```diff
@@ -32,7 +32,7 @@ func (s *GroupMessageStorer) StoreLog(ctx context.Context, event types.Log) LogS
 	}
 
 	// TODO:nm figure out topic structure
-	topic := buildTopic(msgSent.GroupId)
+	topic := BuildGroupMessageTopic(msgSent.GroupId)
```

Comment:
Worth discussing, should we be building this topic on the server, rather than have the client specify it? There is a 'targetTopic' field on the authenticated data at the moment

Reply:
I think so generally. But every byte counts on the smart contract, so I'm OK with stripping it down to the bare minimum there.

We can add some validation to make sure that they match.